chromium_src/chrome/browser/renderer_context_menu/render_view_context_menu.cc

```diff
@@ -634,6 +650,14 @@ void BraveRenderViewContextMenu::BuildAIChatMenu() {
 
   ai_chat_submenu_model_.AddTitleWithStringId(
       IDS_AI_CHAT_CONTEXT_QUICK_ACTIONS);
+
+#if BUILDFLAG(ENABLE_AI_REWRITER)
+  if (ai_rewriter::features::IsAIRewriterEnabled()) {
```

Comment:
I wonder if we want to add more checks such as `params_.selection_text.empty()`

Reply:
Mmmh, I think for now it's fine - eventually the dialog box will pop up with the selected text (if any). The original text will be editable, and then they can modify it with an AI prompt.